scripts/reset_test_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Make sure to give this script execution permission (chmod +x reset_test_db.sh)
```

Comment:
git stores the executable bit in the repo, so no need to mention this

Reply:
Learnt something new!